lekko_client/clients/cached_git_client.py

```diff
@@ -107,3 +109,11 @@ def close(self) -> None:
         if self.watcher:
             self.watcher.stop()  # type: ignore
             self.watcher.join()
+
+
+# Finds or computes the blob sha of a file
+def _get_blob_sha(path: str) -> bytes:  # pragma: no cover
+    sha = blob_from_path_and_stat(path.encode(), os.lstat(path)).id
+    if not isinstance(sha, bytes):
```

Comment:
Yeah it's weird. From their code it looks like `.id` should give bytes correctly or crash due to missing attribute error, but they type it in a way that can technically be "Unknown". This is just a very explicit check to satisfy the type system and in case their implementation ever changes.